commands/audit/sca/common.go

```diff
@@ -66,6 +59,19 @@ func RunXrayDependenciesTreeScanGraph(dependencyTree xrayUtils.GraphNode, techno
 	return
 }
 
+// Infer the status code of SCA Xray scan, must have at least one result, if err occurred or any of the results is `failed` return 1, otherwise return 0.
+func GetScaScansStatusCode(err error, results ...services.ScanResponse) int {
```

Comment:
Question: Shouldn't we have an option that the scan were not performed at all? (nil)

Reply:
we have, it's the default value (nil) only if performed a value will be set

Reply:
we have a calculation at the parser to take that into account (for example at simple-json parser)